fudge-ratio/src/main/webapp/code/utils.js

```diff
@@ -0,0 +1,6 @@
+const get = id => document.getElementById(id);
```

Comment:
so we dont write ``document.getElementById(id)`` and just ``get(id)``

Reply:
Really cool!